src/biogeophys/WaterStateType.F90

```diff
@@ -51,6 +51,10 @@ module WaterStateType
 
      real(r8) :: aquifer_water_baseline                ! baseline value for water in the unconfined aquifer (wa_col) for this bulk / tracer (mm)
 
+     ! Hillslope stream variables
+     real(r8), pointer :: stream_water_lun       (:)   ! landunit volume of water in the streams (m3)
+     real(r8), pointer :: stream_water_depth_lun (:)   ! landunit depth of water in the streams (m3)
```

Comment:
stream_water_depth_lun is a diagnostic variable, and I mainly wanted it for history.  Is it acceptable to have a history variable, without a corresponding model variable?  I'd like to have the option of outputting the depth, instead of having the user need to convert the stream water volume from the history file.   Could I do remove stream_water_depth_lun and do something like this:
 
data1dptr => stream_water_volume_lun(begl:endl)
 do l = begl,endl
    data1dptr(l) = stream_water_volume(l) &
                  /lun%stream_channel_length(l) &
                  /lun%stream_channel_width(l)
 enddo
 call hist_addfld1d (fname=this%info%fname('STREAM_WATER_DEPTH'),  units='m',  &
            avgflag='A', &
            long_name=this%info%lname('depth of water in stream channel (hillslope hydrology only)'), &
            ptr_lunit=data1dptr, l2g_scale_type='natveg',  default='inactive')

Reply:
No, that won't work. It really needs to be a model variable. It's fine to have this as an extra model variable, it should just go in the correct module.